lgc/patch/VertexFetch.cpp

```diff
@@ -897,7 +900,7 @@ bool VertexFetch::needPatchA2S(const VertexInputDescription *inputDesc) const {
 // Checks whether the second vertex fetch operation is required (particularly for certain 64-bit typed formats).
 //
 // @param inputDesc : Vertex input description
-bool VertexFetch::needSecondVertexFetch(const VertexInputDescription *inputDesc) const {
+bool VertexFetch::needSecondVertexFetch(const VertexInputDescription *inputDesc) {
```

Comment:
This looks like a spurious change -- maybe a leftover from a previous version?

Reply:
I think that is right.  I'll undo this change.